_includes/schedule.html

```diff
@@ -5,7 +5,15 @@
 
     <div class="content-wrapper">
 
-        <hr />
+        <div class="col-lg-10 col-lg-offset-1">
+            <h3>CarpentryCon 2022 Schedule</h3>
+            <p class="text-left animated hiding" data-animation="fadeInUp" data-delay="0">
+                <h6>Shortbread liquorice lollipop candy canes bonbon marzipan icing. Pie gingerbread shortbread macaroon jelly topping pastry. Icing cake pie chocolate cake danish cheesecake dragée lemon drops powder. </h6>
```

Comment:
Change text to read: "This calendar lists all of the sessions scheduled for CarpentryCon 2022. The default is a list view of sessions, but you can also toggle to a calendar view. Times will display in your time zone. In the title of each session, R indicates that the session has an attendance cap and requires separate registration in addition to [general registration](https://carpentrycon2022.eventbrite.com/). A link to register is available in the event link. An asterisk (*) indicates that the session is repeated. The additional times available are listed in the event link. ES indicates a Spanish session and EN indicates an English session. ES/EN indicates a session with presentations in both languages. If you have any questions about the program, please email carpentrycon@carpentries.org."

Reply:
I'm on mobile right now and don't have functionality to make this change. The text needs to be entered in the html code in place of the placeholder text.